fs/rc/params_test.go

```diff
@@ -252,7 +252,7 @@ func TestParamsGetBool(t *testing.T) {
 		{false, false, ""},
 		{"true", true, ""},
 		{"false", false, ""},
-		{"fasle", false, "couldn't parse"},
+		{"false", false, "couldn't parse"},
```

Comment:
I assume this misspell was intended, and why the test is now failing.

Reply:
Yes, I wanted to ask about it and eventually forgot. WIll fix it.